chatbot_ner/config.py

```diff
@@ -1,12 +1,12 @@
 from __future__ import absolute_import
 
-import logging
+import structlog
 import os
 
 from elasticsearch import RequestsHttpConnection
 from requests_aws4auth import AWS4Auth
 
-ner_logger = logging.getLogger('chatbot_ner')
```

Comment:
For the sake of backward compatibility, we should retain the old log name, I think it is`ner_ner_log`, you can check in the loglake.

Reply:
the logger name is `chatbot_ner` as it was earlier, however on loglake the it will be `ner_log_log`